src/lib.rs

```diff
@@ -34,6 +34,10 @@ pub enum Workload {
 
 #[cfg(target_os = "linux")]
 mod arch {
+    use indicatif::{ProgressBar, ProgressStyle};
+    use scopeguard;
```

Comment:
```suggestion
```
I think this line may be unnecessary?

Reply:
Thanks! This is OBE since we removed scopeguard altogether, but you were right.